src/azure-cli/HISTORY.rst

```diff
@@ -75,6 +75,14 @@ Release History
 * `az vm availability-set list`: Support showing VM list (#13090)
 * `az vm list-skus`: Fix display problem of table format (#13184)
 
+**IoT**
```

Comment:
We don't directly change this file anymore. Please modify your title and first comment based on the instruction to add history.